src/main/java/hudson/plugins/sshslaves/SSHLauncher.java

```diff
@@ -866,7 +881,7 @@ public Boolean call() throws InterruptedException {
             Boolean res;
             try {
                 res = results.get(0).get();
-            } catch (ExecutionException e) {
+            } catch (Exception e) {
```

Comment:
CancelException is not treated, probably more, it breaks the retries logic, by catching the Exception the CancelException is treated.